### PR TITLE
Fix: Japanese typo

### PR DESCRIPTION
**Pull request by eltociear**
delete duplicate 'の'
correct sentence

### CODE REVIEW

File: ja/contributing/backwards-compatibility.rst [hunk 1]
```diff
@@ -164,7 +164,7 @@ CakePHP をより良くする手助けをしようという場合、機能の追
 
 .. [1] マイナーリリースでコードが破壊される *恐れが* あります。
        詳細は移行ガイドをチェックしてください。
-.. [2] 古いクラス名・メソッド名を利用可能なようの残すことで名前の変更ができます。
+.. [2] 古いクラス名・メソッド名を利用可能に残しておくことで名前の変更ができます。
```
Comment (okinaka) [inline ja/contributing/backwards-compatibility.rst]:
```suggestion
.. [2] 古いクラス名・メソッド名を利用可能なように残しておくことで名前の変更ができます。
```

**Comment by eltociear** [comment]
thanks reviews.

**Comment by eltociear** [comment]
I've fixed document.
Please review again.

**Comment by markstory** [comment]
Thank you 👍

**Pull request merged**